src/fprime/cookiecutter_templates/cookiecutter-fprime-project/hooks/post_gen_project.py

```diff
@@ -12,7 +12,7 @@
 import subprocess
 import sys
 
-DEFAULT_BRANCH = "master"
+DEFAULT_BRANCH = "devel"
```

Comment:
Is it possible to remove this defined-in-multiple-places for this tag?

Reply:
Yeah it will be a little quirky cause we don't simply want to pull out `fprime_branch_or_tag` after the user has set it - as this `DEFAULT_BRANCH` really is a failsafe in case the user input is not a valid branch/tag. And cookiecutter doesn't allow to ingest/push that value into the JSON I believe.
Implementing the less quirky/most verbose way I can think of.

Reply:
Pushed - let me know what your think

Reply:
Great!